docs/contribute/release_checklist.md

```diff
@@ -4,40 +4,155 @@ title: Release checklist
+    +* [Configuration](https://github.com/mozilla-mobile/android-components/v98.0.0/main/.config.yml)
+    +
+    ```
+
+5. Create a commit named `Update to GeckoView [nightly_version].0` that bumps the GeckoView `version` in [Gecko.kt](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt) to the latest `[nightly_version]` build when available in [maven/geckoview-nightly](https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/geckoview-nightly/).
+
+    ```diff
+    diff --git a/buildSrc/src/main/java/Gecko.kt b/buildSrc/src/main/java/Gecko.kt
+    index 991dd702bb5..6f6c4471a26 100644
+    --- a/buildSrc/src/main/java/Gecko.kt
+    +++ b/buildSrc/src/main/java/Gecko.kt
+    @@ -9,7 +9,7 @@ object Gecko {
+        /**
+          * GeckoView Version.
+          */
+    -    const val version = "98.0.20220207065816"
+    +    const val version = "99.0.20220208070047"
+
+        /**
+          * GeckoView channel
+    ```
+
+6. Create a pull request with these 3 commits and land. If this is landed after the scheduled [cron task](https://github.com/mozilla-mobile/android-components/blob/main/.cron.yml#L13) build that will build and bump AC automatically, trigger a manual AC build through the [hook](https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-android-components%2Fnightly). If a manual AC build was needed, trigger the [hook](https://firefox-ci-tc.services.mozilla.com/hooks/project-releng/cron-task-mozilla-mobile-fenix%2Fbump-android-components) to bump
+6. After an hour, follow up by checking if a new `[nightly_version]` AC build is available in [nightly.maven/components](https://nightly.maven.mozilla.org/?prefix=maven2/org/mozilla/components/).
+
+See [https://github.com/mozilla-mobile/android-components/pull/11519](https://github.com/mozilla-mobile/android-components/pull/11519) for an example.
 
 ## After the release
 
-- Send release message to the [Matrix channel](https://chat.mozilla.org/#/room/#android-components:mozilla.org):
+1. Verify that Fenix is receiving the latest `[nightly_version]` AC version in `main`.
+2. Verify that the links in the [changelog.md](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) and new release are working. 
+3. Send release message to the [Matrix channel](https://chat.mozilla.org/#/room/#android-components:mozilla.org):
 ```
 **Android Components 0.17 Release (:ocean:)**
 Release notes: https://mozilla-mobile.github.io/android-components/changelog/
 Milestone: https://github.com/mozilla-mobile/android-components/milestone/15?closed=1
 ```
-- Update version number in repository
-  - [.buildconfig.yml](https://github.com/mozilla-mobile/android-components/blob/main/.buildconfig.yml#L1)
-  - Update the [`version.txt`](https://github.com/mozilla-mobile/android-components/blob/main/version.txt) as well.
-- Create the next milestone
+
+### Minor releases
```

Comment:
I think this chapter we can skip now. Essentially minor releases are following the exact same process now and also happen automatically for GV updates.

Reply:
Removed.